CHANGELOG.md

```diff
@@ -10,10 +10,14 @@
 
 ### ⚠️ Breaking Changes ⚠️
 
-- `uniffi_bindgen`: Removed the `run_main` function.  It's moved to `uniffi::uniffi_bindgen_main` and now unconditionally succeeds rather than return a `Result<()>`
+- `uniffi_bindgen` no longer provides standalone binary.  Having a standalone binary resulted in version mismatches when the `uniffi` version specified in `Cargo.toml` didn't match the `uniffi_bindgen` version installed on the system.  Read [The foreign language bindings](https://mozilla.github.io/uniffi-rs/tutorial/foreign_language_bindings.html) section of the manual for how to set up a `uniffi-bindgen` binary that's local to your workspace.
+- `uniffi_bindgen`: Removed the `run_main` function.  It's moved to `uniffi::uniffi_bindgen_main` and now unconditionally succeeds rather than return a `Result<()>`.
 
 ### What's changed
 
+- The UniFFI crate organization has been significantly reworked:
+  - Projects that use UniFFI for binding/scaffolding generation now only need to depend on the `uniffi` crate and no longer need to depend on `uniffi_bindgen`, `uniffi_build`, etc.
+  - The version numbers for each crate will no longer by kept in sync after this release.  In particular `uniffi` will have breaking changes less often than `uniffi_bindgen` and other crates.  This means that UniFFI consumers that specify their versions like `uniffi = "0.23"` will not need to bump their `uniffi` version as often as before.
```

Comment:
Should we update our own docs at `docs/release-process.md` on how to release individual crates?

Reply:
Yeah, but I actually wasn't sure how the new release process would work now.  I was thinking we could run through a few releases and update the docs as we went.  I'm not really sure though, WDYT?

Reply:
We might as well make a best-effort attempt - I assume you will be making the release, so maybe note what you expect to do, then we can tweak as we go?  (ie, if I had to make it, I wouldn't really know where to start)

Reply:
That makes sense to me, just pushed a commit with my attempt at this.